nixos/modules/services/monitoring/traccar.nix

```
@@ -65,7 +83,7 @@ in
 
   config =
     let
-      configuration = mkConfig cfg.settings;
+      configuration = if cfg.settingsPath != null then cfg.settingsPath else mkConfig cfg.settings;
```

```suggestion
      configuration = if cfg.settingsFile != null then cfg.settingsFile else mkConfig cfg.settings;
```
How did you test this change?

---

And does the `settingsFile` override work? The condition here checks for "settingsPath" which is not defined in this file.

---

Good catch. I must have created this PR with an outdated version of my patch.
Fixed and tested again. Should be fine now.